test/it/pipeline/src/test/java/org/apache/shardingsphere/test/it/data/pipeline/core/job/service/GovernanceRepositoryAPIImplTest.java

```diff
@@ -155,7 +156,26 @@ void assertPersistJobOffsetInfo() {
         assertTrue(actual.isPresent());
         assertThat(actual.get(), is("testValue"));
     }
-    
+
+    @Test
+    void testGovernanceRepositoryAPIImplMethods() {
+        String parentJobId1 = "parentJob123";
+        String checkJobId1 = "checkJob456";
+        governanceRepositoryAPI.persistLatestCheckJobId(parentJobId1, checkJobId1);
+        Optional<String> retrievedCheckJobId1 = governanceRepositoryAPI.getLatestCheckJobId(parentJobId1);
```

Comment:
From Code of Conduct, `retrievedCheckJobId1` could be `actualCheckJobId`

Reply:
Solved!